src_ext/Makefile.packages

```diff
@@ -149,3 +156,8 @@ uutf-pkg-build:
 jsonm-pkg-build:
 	ocaml pkg/pkg.ml build --pkg-name jsonm --pinned false
 	ocamlfind install jsonm _build/pkg/META $(addprefix _build/src/jsonm.,$(EXTS1)) $(addprefix _build/src/jsonm.cm,$(EXTS2)) _build/src/jsonm*.cmx
+
+sha-pkg-build: stublibs
+	dune build @install -p sha
+	dune install -p sha sha
+	cp _build/install/default/lib/stublibs/dllsha_stubs.so $(SITELIB)/stublibs
```

Comment:
This line isn't needed - Dune was always installing it.

Reply:
indeed!